roles/edpm_neutron_dhcp/OWNERS

```diff
@@ -0,0 +1,8 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+approvers:
+  - slawqo
```

Comment:
I actually think you could also add yourself to network-approvers list (separate PR). Just saying.